### ISSUE TITLE
Get-BcArtifactUrl sometimes throws "Service unavailable" 

### ISSUE THREAD

**Issue by jwikman** [microsoft/navcontainerhelper#1097]
A couple of times this week I've got failing jobs because  Get-BcArtifactUrl encounter an "Service unavailable" error when calling Invoke-RestMethod.

Right now I got an error when using artifact 'bcartifacts/sandbox//es/latest'

```
BCArtifact: bcartifacts/sandbox//es/latest
##[error]Invoke-RestMethod : body {font-family:Arial; margin-left:40px; }img  { border:0 none; }#content { margin-left: auto; ma
##[error]rgin-right: auto }#message h2 { font-size: 20px; font-weight: normal; color: #000000; margin: 34px 0px 0px 0px }#messag
e p  { font-size: 13px; color: #000000; margin: 7px 0px 0px0px}#errorref { font-size: 11px; color: #737373; margin-top:
##[error] 41px }Service unavailableOur services aren't available right nowWe're working to restore all services as soon as possi
##[error]ble. Please check back soon.0HTsHXwAAAABcPDjkjRfNQb0bwELod9NdQ1BIMzBFREdFMDQxMQA4ZTU1MDUyNC0zZjkwLTQ5NzUtYTExMS0wODQ2Yz
##[error]BjYTk0M2M=
##[error]At C:\Agent_AS011_01\_work\150\s\NAB DevOps Tools\PowerShell\navcontainerhelper\Misc\Get-BcArtifactUrl.ps1:86 char:21
##[error]+ ... $Response = Invoke-RestMethod -Method Get -Uri "$GetListUrl$nextMarke ...
##[error]+                 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
##[error]    + CategoryInfo          : InvalidOperation: (System.Net.HttpWebRequest:HttpWebRequest) [Invoke-RestMethod], WebExc 
##[error]   eption
##[error]    + FullyQualifiedErrorId : WebCmdletWebResponseException,Microsoft.PowerShell.Commands.InvokeRestMethodCommand
##[error]PowerShell exited with code '1'.

```

If I rerun the pipeline everything works just fine.

Maybe this error should be caught and then have a retry loop with a few seconds of delay?

**Comment by freddydk** [comment]
Yeah, that's why I have the retry in download - if the download isn't successful on cdn - then go to blob store.
Will add that to get-bcartifacturl as well.

**Comment by freddydk** [comment]
Shipped in 0.7.0.14
